sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Create-disbursement-overawards-table.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS sims.disbursement_overawards(
+  id SERIAL PRIMARY KEY,
+  student_id INT NOT NULL REFERENCES sims.students(id),
+  student_assessment_id INT REFERENCES sims.student_assessments(id),
+  disbursement_schedule_id INT REFERENCES sims.disbursement_schedules(id),
```

Comment:
What is the purpose of having student_assessment_id and disbursement_schedule_id both. ?

As disbursement_schedule is connected to student_assessment.

Reply:
Both are optional. The records in the table can be related to an award, which will then provide a `disbursement_schedule_id ` or then be related to the assessment itself when we will have the `student_assessment_id` or both can be null due a `Manual Entry` or `Imported Record`